<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="jenkins-agent-10-25-224-218" name="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" tests="6" failures="0" timestamp="2022-11-09T22:18:49 UTC" time="60.428" errors="0">
  <testcase name="invokeAsyncTestWith_504_And_No_MaxRetryTime" time="0.049" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_502" time="13.055" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_503" time="40.041" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_429" time="5.120" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_429_And_No_MaxRetryTime" time="0.101" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_504" time="2.062" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
</testsuite> <!-- com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest -->
